ui/public/locales/pt_BR.json

```diff
@@ -1554,304 +2031,410 @@
+"label.vm.snapshots": "Snapshot de VM",
 "label.vm.start": "In\u00edcio",
 "label.vm.stop": "Parar",
 "label.vmdisplayname": "Nome de exibi\u00e7\u00e3o da VM",
 "label.vmfs": "VMFS",
 "label.vmfs.datastore": "VMFS datastore",
 "label.vmipaddress": "Endere\u00e7o IP da VM",
-"label.vmlimit": "Limites da Inst\u00e2ncia",
+"label.vmlimit": "Limites de inst\u00e2ncias",
 "label.vmname": "Nome da VM",
 "label.vms": "VMs",
 "label.vms.in.tier": "M\u00e1quinas virtuais em camadas",
 "label.vmstate": "Estado da VM",
-"label.vmtotal": "Total VMs",
+"label.vmtotal": "VMs totais",
+"label.vmware.storage.policy": "Pol\u00edtica de armazenamento do VMWare",
 "label.vmwaredcid": "ID do datacenter VMware",
 "label.vmwaredcname": "Nome do datacenter VMware",
 "label.vmwaredcvcenter": "Vcednter do datacenter VMware",
 "label.vmwarenetworklabel": "Etiqueta de tr\u00e1fego VMware",
 "label.vnmc": "VNMC",
 "label.vnmc.devices": "Dispositivos VNMC",
-"label.volgroup": "Grupo de Volume",
+"label.volgroup": "Grupo de volume",
 "label.volume": "Disco",
 "label.volume.details": "Detalhe do volume",
+"label.volume.empty": "Nenhum volume de dados anexado a esta VM",
+"label.volume.ids": "ID do volume",
 "label.volume.migrated": "Volume migrado",
+"label.volume.volumefileupload.description": "Clique ou arraste o arquivo para esta \u00e1rea para carreg\u00e1-lo",
 "label.volumechecksum": "MD5 checksum",
+"label.volumechecksum.description": "Utilize o hash que voc\u00ea criou no inicio do procedimento de carregamento de volume",
 "label.volumefileupload": "Arquivo local",
-"label.volumegroup": "Grupo de Volume",
-"label.volumelimit": "Limites de Disco",
-"label.volumename": "Nome do Disco",
+"label.volumegroup": "Grupo de volume",
+"label.volumeid": "Volume",
+"label.volumeids": "Volumes a serem removidos",
+"label.volumelimit": "Limites de disco",
+"label.volumename": "Nome do disco",
 "label.volumes": "Discos",
 "label.volumetotal": "Disco",
 "label.vpc": "VPC",
-"label.vpc.id": "VPC ID",
+"label.vpc.id": "ID da VPC",
+"label.vpc.offering.access": "Oferta de acesso da VPC",
 "label.vpc.offering.details": "Detalhes da oferta VPC",
 "label.vpc.offerings": "Ofertas VPC",
 "label.vpc.router.details": "Detalhes do roteador da VPC",
-"label.vpc.virtual.router": "Roteador Virtual VPC",
+"label.vpc.virtual.router": "Roteador virtual VPC",
 "label.vpcid": "VPC",
 "label.vpclimit": "Limites VPC",
 "label.vpcname": "VPC",
 "label.vpcoffering": "Oferta VPC",
+"label.vpcofferingid": "Oferta VPC",
+"label.vpctotal": "VPCs totais",
 "label.vpn": "VPN",
 "label.vpn.connection": "Conex\u00e3o VPN",
 "label.vpn.gateway": "Gateway de VPN",
+"label.vpn.users": "Usu\u00e1rios da VPN",
+"label.vpncustomergateway": "Endere\u00e7o de IP do gateway remoto",
+"label.vpncustomergateway.cidrlist": "A lista de CIDR de guest separada por v\u00edrgula das subredes remotas.",
+"label.vpncustomergateway.esplifetime": "A vida \u00fatil do security association na fase-2 em segundos",
```

Comment:
You can make it `Associação de Segurança na fase-2` instead of `Security Association an fase-2`.
Same goes for the line below regarding "fase-1".

As far as I know, that's how it is used in Brazil (at least in the academic world where we need to translate everything to Portuguese :stuck_out_tongue:)